Source/Scene/ModelExperimental/ModelFeature.js

```diff
@@ -10,7 +10,7 @@ import defined from "../../Core/defined.js";
  * Modifications to a <code>ModelFeature</code> object have the lifetime of the model.
  * </p>
  * <p>
- * Do not construct this directly. Access it through {@link ModelFeatureTable#getFeature}, {@link Cesium3DTileContent#getFeature} or
+ * Do not construct this directly. Access it through {@link Cesium3DTileContent#getFeature} or
```

Comment:
> Access it through {@link Cesium3DTileContent#getFeature}

This part may not be relevant anymore. `Cesium3DTileContent#getFeature` will return a `Cesium3DTileFeature` not a `ModelFeature`. @sanjeetsuhag can you confirm?

Reply:
Yes, that is correct. If part of 3D Tiles, `Cesium3DTileFeatures` will be returned. [Sandcastle](http://localhost:8080/Apps/Sandcastle/index.html#c=bVJNj9MwEP0rQ06ptHKEuNFsBWS7YkURh0bsJRJy7cnW4NiRPcnSRf3v2PlATdlDYs/Hm5n3PAV61TVs+7tFpxo0xPU9cuoceoaGHzR+tRL1ZRxugVyH68oIazxBr/AZXfAafIZirPd98KVVIga7sAGoDLoqWQXcjCSl0SMtoePx7q4cg+mfygB0Tr+HKsn2LQqf3XHi2SIv+8RJHFHO56Oio+1osMrIIpt6sZ/emiqpzHkcZByeeYEGWRsoKlJ94M6lTCdITJzSXqxtSnsZCCHuwKNGQSgn6ZaVHaLZt1zgtg/6feZG6uhHejBtRx8FKWvSujPDBazZYU2FVuJXGpk3tseoe2VWMEgR+7Uh+q9bkG/JIkJnGGutV7FwHBZA1ZC+mYSWWIcnkemi2GrqAuAw2GZAneMvflc8Q+cFeD3nxee1Gpm2T+kVJs5xvpkfe3+lTnlqke229+WPYvdQfFmtk5sk93TSuBmnAvigmtY6iiuRMpYRNq3mFFbg0IVRiAnvR64AeXYJzaXqQcnbV5YShObeh0jdab1XL1glmzwL+f9BteVSmadvPTrNTzHt+HazG52MsTwL5utIslYfuLuq/Bc)